tests/integration/test_agentd/test_agentd_server_address_configuration.py

```diff
@@ -0,0 +1,97 @@
+import os
+import subprocess
+import pytest
+from time import sleep
+
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.file import truncate_file
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.services import control_service
+import wazuh_testing.agent as agent
+
+TIMEOUT = 30
+
+# Marks
+pytestmark = pytest.mark.tier(level=0)
```

Comment:
This test is only for agents.
```suggestion
pytestmark = [pytest.mark.tier(level=0), pytest.mark.agent]
```